pkg/log/rotate_test.go

```diff
@@ -182,11 +186,9 @@ func TestFileRotate(t *testing.T) {
 		assert.NoError(t, os.Remove(rotatedLogFile))
 	}()
 
-	require.NoError(t, err)
-
 	// Mock the date for filename rename
 	file.getTimeFn = func() time.Time {
-		return time.Date(2022, time.January, 1, 01, 23, 45, 0, time.Local)
+		return time.Date(2022, time.January, 1, 0o1, 23, 45, 0, time.Local)
```

Comment:
☝️  `0o1` --> `01`

Reply:
gofumpt does this, otherwise linter fails